### PR TITLE
File Added: Assignment 7 and 8

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

File: akash1/HTML/ASSIn7/login.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>login</title>
+        <link rel="stylesheet" href="style.css" />
+
```
Comment (shivling123) [inline akash1/HTML/ASSIn7/login.html]: remove extra space

File: akash1/HTML/ASSIn7/personal.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    <input type="checkbox" id="HTML" name="skillset" value="HTML" />
+                    <label for="vehicle1">HTML</label>
+                    <input type="checkbox" id="Javascript" name="skillset" value="Javascript" />
+                    <label for="vehicle2">Javascript</label>
+                    <input type="checkbox" id="java" name="skillset" value="java" />
+                    <label for="vehicle3">Java</label>
+                    <input type="checkbox" id="oracle" name="skillset" value="oracle" />
+                    <label for="vehicle3">oracle</label><br />
+                    <label for="highest Qualification">Highest Qualification:</label>
+                    <select name="highest Qualification" id="Qualification">
```
Comment (shivling123) [inline akash1/HTML/ASSIn7/personal.html:66]: indentation is not proper format code propper 
after select tag

File: akash1/HTML/ASSIn7/personal.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    <option value="computer science">Diploma In Computer science</option>
+                    <option value="science">science graduate</option>
+                    <option value="engg">Bachelor in Engg</option>
+                    <option value="master">Master in Engg</option>
+                    <option value="other">other</option>
+                    </select><br>
+                <label for="myfile">Upload Resume:</label>
```
Comment (shivling123) [inline akash1/HTML/ASSIn7/personal.html]: check indentation

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes as per comment anduplod again asap

**Review by Pranjali1999 — APPROVED** [review]

**Review by vinod2406 — APPROVED** [review]

File: akash1/HTML/ASSIn7/personal.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                <div class="block2">
+                    <h2>Personal details</h2>
+                </div>
+                <hr />
+                <div class="block4">
+                    <label>First Name:</label>
+                    <input type="text" name="fname" placeholder="First name" /><br />
+                    <label>Last Name:</label>
+                    <input type="text" name="lname" placeholder="Last name" /><br />
+                    <label for="gender">Gender:</label>
```
Comment (preetipie22) [inline akash1/HTML/ASSIn7/personal.html:35]: only  1 button should be checked at a time

**Review by preetipie22 — CHANGES_REQUESTED** [review]
Please make the requested change

**Review by Akash-ing — APPROVED** [review]

File: akash1/HTML/ASSIn7/personal.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    <label for="highest Qualification">Highest Qualification:</label>
+                    <select name="highest Qualification" id="Qualification">
+                        <option value="computer science">Diploma In Computer science</option>
+                        <option value="science">science graduate</option>
+                        <option value="engg">Bachelor in Engg</option>
+                        <option value="master">Master in Engg</option>
+                        <option value="other">other</option>
+                    </select><br>
+                    <label for="myfile">Upload Resume:</label>
+                    <input type="file" id="myfile" name="myfile" />
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/personal.html:74]: remove the / at the end of line

File: akash1/HTML/ASSIn7/profession.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="block6">
+            <div class="block4">
+                <h1>Professional Details</h1>
+            </div>
+            <hr />
+            <div class="block5">
+                <h3>Work Experience</h3>
+                <hr />
+                <label for="company"><h3>Company</h3></label>
+                <input type="text" name="company" /><br />
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/profession.html:29]: remove the / at the end of line

File: akash1/HTML/ASSIn7/profession.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+                <label for="jobtitle"><h3>Job Title</h3></label>
+                <input type="text" name="jobtitle" placeholder="Job Title" /><br />
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/profession.html:31]: remove the / at the end of line

File: akash1/HTML/ASSIn7/style.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style.css]: remove extra line

File: akash1/HTML/ASSIn7/style.css [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+li {
+    float: left;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style.css]: remove extra line

File: akash1/HTML/ASSIn7/style.css [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+li a {
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style.css]: remove extra line

File: akash1/HTML/ASSIn7/style.css [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+li a:hover {
+    background-color: #111;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style.css]: remove extra line

File: akash1/HTML/ASSIn7/style.css [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+    background-image: url("23.jpg");
+    height: 500px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family: "Baloo Bhaina 2", cursive;
+    margin: 0px;
+    padding: 0px;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style.css]: remove extra line

File: akash1/HTML/ASSIn7/style1.css [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+    border-radius: 10px;
+}
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style1.css]: remove line

File: akash1/HTML/ASSIn7/style1.css [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+li {
+    float: left;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style1.css]: remove extra line

File: akash1/HTML/ASSIn7/style1.css [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+li a {
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style1.css]: remove extra line

File: akash1/HTML/ASSIn7/style2.css [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+    background-color: black;
+}
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style2.css]: remove extra line

File: akash1/HTML/ASSIn7/style2.css [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+li {
+    float: left;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style2.css]: remove extra line

File: akash1/HTML/ASSIn7/style2.css [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+li a {
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn7/style2.css]: remove extra line

File: akash1/HTML/ASSIn8/style4.css [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+.block1 {
+    padding: 24px 45px 0px 36px;
+}
+.block2 {
+    padding: 87px 45px 0px 36px;
+}
+.block3 {
+    padding: 72px 45px 0px 36px;
+}
+
```
Comment (sourabh513) [inline akash1/HTML/ASSIn8/style4.css:22]: remove extra line

File: akash1/HTML/ASSIn7/profession.html (hunk 7 above)
Comment (akash19972306) [reply akash1/HTML/ASSIn7/profession.html:31]: it is automatically come when identitatione apply

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**